x-pack/plugins/canvas/public/components/workpad_page/event_handlers.js

```diff
@@ -137,6 +137,16 @@ const handleKeyDown = (commit, e, isEditable, remove) => {
   }
 };
 
+const handleKeyPress = (commit, e, isEditable) => {
```

Comment:
Is there a reason you're not handling this in the `handleKeyDown` handler? My understand is that both handlers will be called in most cases, with alt, ctrl, shift, and meta being the exception.

> **MDN**: The keydown event is fired when a key is pressed down. Unlike the keypress event, the keydown event is fired for keys that produce a character value and for keys that do not produce a character value.

Reply:
Yes it could technically work both ways (actually, keyPress will activate on the upward trajectory), it was simply a question of showing intent - one is concerned with signaling intent with a keypress gesture - `'G'` or `'U'` here - while the other conveys up / down events.

Reply:
Yes, keypress does ignore those keys. My point was that `onKeyDown` is always going to be called, and that internal event would always be triggered, even though those specific keys are handled differently. Sounds like it shouldn't cause problems though.